### ISSUE TITLE
[Bug] Long texts are not shown

### ISSUE THREAD

**Issue by Terkyz** [0xAdiyat/ToDo-WishFlow#3]
As the title mentions, by adding a lot of text (a bunch of words in reality) to the point that the text slips to continue reading, after confirming the creation of the task, the text of the task that is shown are simply the first words and cannot be slipped to read the other content.

**Comment by 0xAdiyat** [comment]
> As the title mentions, by adding a lot of text (a bunch of words in reality) to the point that the text slips to continue reading, after confirming the creation of the task, the text of the task that is shown are simply the first words and cannot be slipped to read the other content.

Can I have a screenshot or screen record of the problem? Thank you.

**Comment by Terkyz** [comment]
Yep, [here](https://imgur.com/a/RvaFNa4) you can see the problem.
